src/translate.directive.ts

```diff
@@ -27,6 +28,13 @@ export class TranslateDirective implements AfterViewChecked, OnDestroy {
                 this.checkNodes(event.translations);
             });
         }
+
+        // subscribe to onDefaultLangChange event, in case the default language changes
+        if(!this.onDefaultLangChangeSub) {
+            this.onDefaultLangChangeSub = this.translateService.onDefaultLangChange.subscribe((event: DefaultLangChangeEvent) => {
+                this.checkNodes(event.translations);
```

Comment:
Seems like I forgot to add tests for the pipe as well when the default lang changes. Added those to check if it does work there without the translations appended to the updateValue call and it does work there. Maybe an error in the checkNodes function in the directive?